pkg/test/framework/components/istio/operator.go

```diff
@@ -226,6 +227,15 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 				if err := deployControlPlane(i, cfg, cluster, iopFile); err != nil {
 					return fmt.Errorf("failed deploying control plane to cluster %s: %v", cluster.Name(), err)
 				}
+
+				if cfg.ExposeIstiod {
```

Comment:
Do we actually need this flag? I think we can figure it out based on whether or not there are remote clusters in the control plane topology. Or maybe we keep it simple and just always expose istiod on the primary clusters?

Reply:
Yes, at least for now, because some integration tests disable ingressGateways, and without this those tests fail. After I add to the default list of ports, I could probably remove this

Reply:
(By yes I mean something conditional is needed. Could potentially do something like see whether there's an ingress Service before trying to patch it. But when I remove the patch, I can remove this flag too. Also, some tests need this that aren't multicluster, like for VMs)

Reply:
Yeah, I think that's better ... just always patch the ingress if there is one.